test/builder/eventlistener_test.go

```diff
@@ -46,6 +50,92 @@ func TestEventListenerBuilder(t *testing.T) {
 				),
 			),
 		},
+		{
+			name: "Status hostname",
+			normal: &v1alpha1.EventListener{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "name",
+					Namespace: "namespace",
+				},
+				Status: v1alpha1.EventListenerStatus{
+					Configuration: v1alpha1.EventListenerConfig{
+						Hostname: "hostname",
+					},
+				},
+			},
+			builder: EventListener("name", "namespace",
+				EventListenerStatus(
+					EventListenerConfig("hostname"),
+				),
+			),
+		},
+		{
+			name: "One Condition",
+			normal: &v1alpha1.EventListener{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "name",
+					Namespace: "namespace",
+				},
+				Status: v1alpha1.EventListenerStatus{
+					Status: duckv1beta1.Status{
+						Conditions: []apis.Condition{
+							apis.Condition{
+								Type:    v1alpha1.ServiceExists,
+								Status:  corev1.ConditionTrue,
+								Message: "Service exists",
+							},
+						},
+					},
+				},
+			},
+			builder: EventListener("name", "namespace",
+				EventListenerStatus(
+					EventListenerCondition(
+						v1alpha1.ServiceExists,
+						corev1.ConditionTrue,
+						"Service exists", "",
```

Comment:
nit: Please add a `reason` in these tests. I think we want to verify that the builder fills all of the required `EventListenerCondition` fields.

Reply:
I just added test cases to ensure the builders match the expected output. Are you referring to an additional field in the anonymous struct?